scripts/database/pg/migrations/030.do.remove-unused-dojolead-column.sql

```diff
@@ -0,0 +1,19 @@
+DO $$
+    BEGIN
+      BEGIN
+        ALTER TABLE cd_dojoleads DROP online_sessions;
+      EXCEPTION
+        WHEN others THEN RAISE NOTICE 'Unhandled error: % %', SQLERRM, SQLSTATE;
+      END;
+      BEGIN
+        ALTER TABLE cd_dojoleads DROP safeguarding_sponsor_name;
+      EXCEPTION
+        WHEN others THEN RAISE NOTICE 'Unhandled error: % %', SQLERRM, SQLSTATE;
+      END;
+      BEGIN
+        ALTER TABLE cd_dojoleads DROP safeguarding_sponsor_email;
+      EXCEPTION
+        WHEN others THEN RAISE NOTICE 'Unhandled error: % %', SQLERRM, SQLSTATE;
+      END;
+    END;
+$$
```

Comment:
Discovered that these columns are not needed.  The data is stored in the 'application' json column and copied over to cd_dojos:

```
cp-dojos-development=# select application->'venue'->>'onlineSessions' as onlineSessions, application->'venue'->>'safeguardingSponsorEmail' as safeguardingSponsorEmail from cd_dojoleads order by updated_at desc limit 1;
 onlinesessions | safeguardingsponsoremail
----------------+---------------------------
 1              | glen.pike@raspberrypi.org
(1 row)
```